content_creator.py

```diff
@@ -146,9 +148,23 @@ def copy_test_files(bundle_test):
             shutil.copyfile(path, os.path.join(bundle_test, os.path.basename(path)))
 
 
-def main(circle_artifacts):
-    print('Starting to create content artifact...')
+def update_content_version(key, new_value):
```

Comment:
Should add try/except in case something here fails, print user friendly message that explains what went wrong and where to look for.